pkg/ccl/changefeedccl/enriched_source_provider.go

```diff
@@ -316,3 +330,5 @@ func init() {
 		Optional: true,
 	}
 }
+
+const originCockroachDB = "cockroachdb"
```

Comment:
nit: would prefer to see this either at the top of the file or just above `newEnrichedSourceProvider`